src/ParallelAlgorithms/Interpolation/Protocols/ComputeTargetPoints.hpp

```diff
@@ -0,0 +1,82 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+struct DataVector;
+/// \endcond
+
+namespace intrp::protocols {
+/*!
+ * \brief A protocol for the type alias `compute_target_points` found in an
+ * InterpolationTargetTag.
+ *
+ * \details A struct conforming to the `ComputeTargetPoints` protocol must have
+ *
+ * - a type alias `is_sequential` that is either `std::true_type` or
+ *   `std::false_type` which indicates if interpolations depend on previous
+ *   interpolations' results
+ *
+ * - a type alias `frame` that denotes the frame the target points are computed
+ *   in
+ *
+ * - a function `points` with the following signature
+ *
+ * \snippet Helpers/ParallelAlgorithms/Interpolation/Examples.hpp points
```

Comment:
(optional) If you're changing other things, I suggest that you move this snippet to the bottom of these docs and expand it to show the entire example implementation of the protocol. Something like: "Here's an example of a class that conforms to this protocol: \snippet ...". I feel like that's more helpful and gives more context than small snippets of code. The same suggestion applies to the other places in this PR where you include snippets of individual function signatures.